python/ray/serve/tests/test_deployment_version.py

```diff
@@ -269,6 +269,75 @@ def test_placement_group_options():
     assert v6.requires_actor_restart(v7)
 
 
+def test_requires_actor_restart():
```

Comment:
since I merged my PR now, can you add placement group-related test cases here? (I added some asserts above, but might as well put them in one place)

Reply:
Sure, I added some test cases to `test_requires_actor_restart` (left your original tests as is)